packages/neuron-ui/src/components/Providers/index.tsx

```diff
@@ -6,7 +6,7 @@ import ModalContext, { initModal, modalReducer, MODAL_ACTION_TYPES } from '../..
 import SettingsContext, { initSettings } from '../../contexts/settings'
 
 import { ipcRenderer } from '../../utils/ipc'
-import { IPC_CHANNEL } from '../../utils/const'
+import { IPCChannel } from '../../utils/const'
```

Comment:
Discussion: is it better to name it `Channel` instead of `IPCChannel`. First: the UI app might not have to know it's an `IPC` channel (which is implementation detail), second: we know `Channel` means communication mechanism and the `IPC` prefix is somewhat unnecessary.

Reply:
I will update it in another PR